cosmic_ray/cli.py

```diff
@@ -61,7 +61,7 @@ def handle_baseline(args):
         print(''.join(work_item.data))
         return 2
 
-    return os.EX_OK
+    return 0
```

Comment:
Please use the new `cosmic_ray.exit_codes` module for the exit codes. See commit 56708ff1b6c9608070088327fd3e305d50284e0d for the details.